management_interface/management_interface/forms.py

```diff
@@ -0,0 +1,33 @@
+from django import forms
+from .models import CareProviderLocation, RegisteredManager, CareRecipient
+from .utils import validated_email_domain
+
+
+class CareProviderLocationForm(forms.ModelForm):
+    class Meta:
+        model = CareProviderLocation
+        exclude = ["id", "created_at", "updated_at"]
+
+    def clean_email(self):
+        email = self.cleaned_data["email"]
```

Comment:
One thing I think we *do* want is whitespace stripping.  It's just obnoxious to accept `"me@nhs.net"` but not `"me@nhs.net "`.

Reply:
So now clean_email() is removing unnecessary spaces only.